backend/src/test/java/com/cruru/util/fixture/MemberFixture.java

```diff
@@ -1,9 +1,16 @@
 package com.cruru.util.fixture;
 
 import com.cruru.member.domain.Member;
+import com.cruru.member.domain.MemberRole;
 
 public class MemberFixture {
 
-    public static final Member DOBBY = new Member("dobby@email.com", "newPassword214!", "01011111111");
-    public static final Member RUSH = new Member("rush@email.com", "newPassword214!", "01022222222");
+    public static final Member ADMIN = new Member(1000000L, "admin@email.com",
+            "$2a$10$rG0JsflKdGcORjGFTURYb.npEgtvClK4.3P.EMr/o3SdekrVFxOvG", "01011111111", MemberRole.ADMIN); // password 원문: qwer1234
+    public static final Member DOBBY = new Member("dobby@email.com",
+            "$2a$10$RKBwn5Sa7EO0lYaZqU1zSupNbPJ5/HOKcI7gNb9c2q.TiydBHUBQK",
+            "01011111111"); // password 원문: newPassword214!
+    public static final Member RUSH = new Member("rush@email.com",
+            "$2a$10$RKBwn5Sa7EO0lYaZqU1zSupNbPJ5/HOKcI7gNb9c2q.TiydBHUBQK",
+            "01022222222"); // password 원문: newPassword214!
```

Comment:
```suggestion
    public static Member dobby() {
        return new Member("dobby@email.com",
                "$2a$10$RKBwn5Sa7EO0lYaZqU1zSupNbPJ5/HOKcI7gNb9c2q.TiydBHUBQK",
                "01011111111"); // password 원문: newPassword214!
    }

    public static Member rush() {
        return new Member("rush@email.com",
                "$2a$10$RKBwn5Sa7EO0lYaZqU1zSupNbPJ5/HOKcI7gNb9c2q.TiydBHUBQK",
                "01022222222"); // password 원문: newPassword214!
    }
```
이 방식으로 하시면 테스트 안터집니당. 아마 메모리 관련 문제이지 않을까 싶네요. id가 공유된다던지,, 그러면 현 상황에서 ADMIN도 필요없겠죠?

Reply:
오 그럴수 있겠네요.. 정확한 원인은 더 파악해봐야겠네요. 해당 사항 반영하려니까 기존 테스트 싹 바꿔야해서. 이건 새로운 이슈로 해결할게요.